kakao/src/main/kotlin/com/agoda/kakao/picker/KDatePicker.kt

```diff
@@ -0,0 +1,30 @@
+package com.agoda.kakao.picker
+
+import android.widget.DatePicker
+import androidx.test.espresso.contrib.PickerActions
+import com.agoda.kakao.common.views.KBaseView
+import com.agoda.kakao.text.KButton
+
+/**
+ * View for interact with default date picker
+ *
+ * @see DatePicker
+ */
+class KDatePicker : KBaseView<KDatePicker>({ isInstanceOf(DatePicker::class.java) }) {
+    val action = KButton {
+        isDescendantOfA { isInstanceOf(DatePicker::class.java) }
+        withId(android.R.id.button1)
+    }
+
+    fun setDate(year: Int, month: Int, day: Int, apply: Boolean = true) {
+        perform {
+            PickerActions.setDate(year, month, day)
```

Comment:
This does nothing, since `perform` is infix function on the `KBaseView`.
Maybe `view.perform(...)` or `act { ... }` is what you're looking for?